### PR TITLE
Update Nats module docs

### PR DESCRIPTION
**Pull request by skatsaounis**
This PR adds an image of `Nats` module dashboard introduced by #10235.

Co-Authored-By: Christos Markou <chrismarkou92@gmail.com>
Co-Authored-By: Michael Katsoulis <michaelkatsoulis88@gmail.com>

### CODE REVIEW

**Comment by skatsaounis** [comment]
@ruflin @MichaelKatsoulis @ChrsMark

File: metricbeat/module/nats/_meta/docs.asciidoc [hunk 1]
```diff
@@ -1,10 +1,16 @@
-This is the nats module.
-
-This is the Nats Module. The Nats module uses http://localhost:8222 APIs to collect metrics.
+This is the Nats module. The Nats module uses http://localhost:8222 APIs to collect metrics.
```
Comment (ruflin) [inline metricbeat/module/nats/_meta/docs.asciidoc]: Should we also link here to the Nats (monitoring docs) for some background?

Comment (skatsaounis) [reply metricbeat/module/nats/_meta/docs.asciidoc]: on its way

**Comment by ChrsMark** [comment]
Related to: https://github.com/elastic/beats/issues/10071

**Pull request merged**